### DISCUSSION TITLE
【求助】在空洞里找到一瓶标签已被腐蚀的物质

### DISCUSSION THREAD

**Discussion by Omituanz** [share121/inter-knot#2610] [其他]
![ezgif-4-5a5ac53f97](https://github.com/user-attachments/assets/e59581ef-014a-4971-801d-8fc76ec37ec3)
标签上只有这几个字：无色或淡黄色油状液体，有强酸味，能与水、醇、醚相混溶，且不溶于三氯甲烷和二硫化碳。有没有懂化学的绳匠分析一下<img src="https://bbs-static.miyoushe.com/static/2023/11/16/b7b6eb7ebdc9f63ba6365e7e8d6b5fd7_5912345410510420967.png" alt="安比-啊···" width="75" >

**Comment by himetuki** [comment]
推测：可食用

**Comment by yytyfj** [comment]
看起来像甘油，但是甘油没有味道

**Comment by yivitte** [comment]
这个委托我接了！

**Comment by SrQingChen** [comment]
这个委托我接了~带回家炒个菜看看是不是白醋(*^▽^*)

**Comment by LShanShi** [comment]
这个委托我接了!

**Comment by juequl** [comment]
这个委托太危险，建议叫基金会来会帮忙！

**Comment by Kiiro-Elysia** [comment]
这个委托我接了！

**Comment by RaphaelHider** [comment]
谁家标签会写这些东西啊喂，有点像小明不记得肇事车牌号但是记住了车牌号是质数（恼）

**Comment by lllliufeng** [comment]
杏仁水（

**Comment by chihung3927** [comment]
猜测是H2O
